### PR TITLE
Added recipe for yahoo mail.

### PR DESCRIPTION
**Pull request by vraravam**

### CODE REVIEW

File: recipes/yahoo-mail/package.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
+  "id": "yahooMail",
+  "name": "yahoo-mail",
```
Comment (vantezzen) [inline recipes/yahoo-mail/package.json:3]:
```suggestion
  "name": "Yahoo Mail",
```

File: recipes/yahoo-mail/package.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
+{
+  "id": "yahooMail",
```
Comment (vantezzen) [inline recipes/yahoo-mail/package.json:2]:
```suggestion
  "id": "yahoo-mail",
```

**Review by vantezzen — CHANGES_REQUESTED** [review]
Thank you for taking the time to add your Yahoo Mail recipe to the Ferdi recipe repository.

The recipe looks good, but I have two suggestions for your package.json:
- You should change the "name" in the package.json to "Yahoo Mail" instead of "yahoo-mail". The attribute is the name shown inside Ferdi
- You should change the "id" to "yahoo-mail". The ID should be the same as the name of the folder the recipe is the (i.e. you named the folder `recipes/yahoo-mail`, thus the ID should be `yahoo-mail`)

File: recipes/yahoo-mail/package.json (hunk 2 above)
Comment (vraravam) [reply recipes/yahoo-mail/package.json]: tx @vantezzen  - changed the 2 fields as per your comments.

File: recipes/yahoo-mail/package.json (hunk 1 above)
Comment (vraravam) [reply recipes/yahoo-mail/package.json]: Done

**Comment by vantezzen** [comment]
Thank you for your contribution! The recipe will be available in the next version of Ferdi.

**Pull request merged**